libs/lib-client/installer/src/lib/installer.service.ts

```diff
@@ -0,0 +1,11 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { ActivitySchema } from '@flogo-web/core';
+
+@Injectable()
+export class InstallerService {
+  contribInstalled = new EventEmitter<ActivitySchema>();
```

Comment:
As per [`EventEmitter` docs](https://angular.io/api/core/EventEmitter) ("Use in directives and components to emit custom events") and [Angular examples about services ](https://angular.io/guide/component-interaction#parent-and-children-communicate-via-a-service) I think we should avoid using EventEmitter and use `Subject` from `rxjs`, also for consistency with the rest of the services in the app.

```ts
contribInstalled$ = new Subject<ActivitySchema>();

// ...

this.contribInstalled$.next(contribDetails);
```

Reply:
Changed EventEmitter to Subject